### PR TITLE
Fix row-thread mapping for `sequential` workloads

### PR DESCRIPTION
**Pull request by vponomaryov**
Before this change scylla-bench was distributing rows among threads correctly only when number of partitions was divisible by number of threads without remainder.

A bit worse case was appearing when there was division remainder.

And the worst case was when number of threads was bigger than number of partitions. In this case last thread was handling all of the rows and all of other threads were idle.

So, fix it by spreading rows evenly among all of the threads as much as possible.

Closes: #138

### CODE REVIEW

**Review by CodeLieutenant — APPROVED** [review]

File: main.go [hunk 1]
```diff
@@ -143,15 +144,16 @@ func PrepareDatabase(session *gocql.Session, replicationFactor int) {
...
-			thisSize = pksPerThread
+		totalRowCount := partitionCount * clusteringRowCount
+		currentThreadId := int64(threadId)
+		rowCount, rowRemainder := totalRowCount/int64(concurrency), totalRowCount%int64(concurrency)
+		additionalRows := int64(0)
+		if currentThreadId < rowRemainder {
+			additionalRows = 1
 		}
-		return NewSequentialVisitAll(thisOffset+partitionOffset, thisSize, clusteringRowCount)
+		currentRemainderPartialOffset := int64(math.Min(float64(rowRemainder), float64(currentThreadId)))
```
Comment (dkropachev) [inline main.go:154]:
```suggestion
		additionalRows := int64(0)
		currentRemainderPartialOffset := int64(rowRemainder)
		if currentThreadId < rowRemainder {
			additionalRows = 1
			currentRemainderPartialOffset = currentThreadId
		}
```

**Review by dkropachev — APPROVED** [review]
Small nit, rest looks good.

File: main.go (hunk 1 above)
Comment (vponomaryov) [reply main.go]: Done

**Review by fruch — APPROVED** [review]
LGTM

**Review by dkropachev — APPROVED** [review]

**Pull request merged**